acq4/devices/PatchPipette/statemanager.py

```diff
@@ -86,7 +86,10 @@ def getProfileConfig(cls, name):
 
     @staticmethod
     def buildPatchProfilesParameters():
-        params = [ProfileParameter(profile) for profile in PatchPipetteStateManager.listProfiles()]
+        params = []
+        for profile in PatchPipetteStateManager.listProfiles():
+            params.append(ProfileParameter(profile))
+            Qt.QApplication.processEvents()
```

Comment:
Why?

Reply:
with pg.BusyCursor
with pg.ProgressDialog(maximum=len(profiles)) as dlg:
    for p in profiles:
        ....
        dlg += 1